tests/integration/piv_worker_test.py

```diff
@@ -115,7 +114,8 @@ def launch_piv_worker(pg_docker_fixture):
         # re-init the db by overriding the db_url
         reinit_flask_db(pg_docker_fixture)
 
-        contributor = get_contributor_by_connector_type(PIV_CONTRIBUTOR_ID, ConnectorType.piv.value)
+        contributors = Contributor.find_by_connector_type(ConnectorType.piv.value)
+        contributor = [c for c in contributors if c.id == PIV_CONTRIBUTOR_ID][0]
```

Comment:
You can probably do a bit more pythonic, like:
```suggestion
        contributor = next(filter(lambda c: c.id == PIV_CONTRIBUTOR_ID, contributors))
```